src/NServiceBus.Core/Pipeline/Outgoing/OutgoingPublishContext.cs

```diff
@@ -1,29 +1,30 @@
 ﻿namespace NServiceBus.OutgoingPipeline
 {
     using NServiceBus.Pipeline;
+    using PublishOptions = NServiceBus.PublishOptions;
 
     /// <summary>
     /// Pipeline context for publish operations.
     /// </summary>
-    public class OutgoingPublishContext : BehaviorContext
+    public class OutgoingPublishContext : OutgoingContext
     {
         /// <summary>
-        /// The message to be published.
-        /// </summary>
-        public OutgoingLogicalMessage Message { get; private set; }
-
-        /// <summary>
         /// Initializes the context with a parent context.
         /// </summary>
         public OutgoingPublishContext(OutgoingLogicalMessage message, PublishOptions options, BehaviorContext parentContext)
             : base(parentContext)
         {
             Message = message;
-            Guard.AgainstNull("parentContext", parentContext);
-            Guard.AgainstNull("message", message);
-            Guard.AgainstNull("options", options);
+            Guard.AgainstNull(nameof(parentContext), parentContext); // TODO DanielTim : Is this needed here
```

Comment:
@timbussmann I'm going to remove the Mapper changed tomorrow and
do context.Builder to keep this PR focused and work on the locator things
in a different low prio PR

-- 
*#NSBcon 2015 • The "All About NServiceBus" conference is back *
*• Learn more <http://go.particular.net/l/27442/2015-08-21/5d4wzp> →*